lib/ansible/modules/user.py

```diff
@@ -89,7 +89,7 @@
             - On macOS systems, this value has to be cleartext. Beware of security issues.
             - To create a disabled account on Linux systems, set this to C('!') or C('*').
             - To create a disabled account on OpenBSD, set this to C('*************').
-            - See U(https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-generate-encrypted-passwords-for-the-user-module)
+            - See R(user password,user_passwords)
```

Comment:
```suggestion
            - See L(this FAQ entry,https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-generate-encrypted-passwords-for-the-user-module)
```

Reply:
@acozine Oh wow. Okay, thanks Alicia!

Reply:
where are we with this one? Someone else also opened a PR to fix it so it seems a common problem users are seeing